src/utils/routePathsConfig.tsx

```diff
@@ -30,25 +31,32 @@ import SendWrappedDecryptionKey from "../pages/pki/SendWrappedDecryptionKey";
 import SetupPki from "../pages/pki/SetupPki";
 import UploadCert from "../pages/pki/UploadCert";
 
-export interface routePath {
+export interface RoutePath {
   path: string;
   label: string;
   component?: JSX.Element;
 }
-export interface routePaths {
-  [key: string]: routePath[];
+export interface RoutePaths {
+  [key: string]: RoutePath[];
 }
 
-export const routePathsConfig: routePaths = {
+export const topSectionTitles = [
```

Comment:
there were other ways to restore the old titles ofc but this seemed quite simple and straightforward honestly